python/test/unit/fem/test_nonlinear_assembler.py

```diff
@@ -301,7 +301,7 @@ def boundary(x):
 
     snes.getKSP().setType("preonly")
     snes.getKSP().getPC().setType("lu")
-    snes.getKSP().getPC().setFactorSolverType("mumps")
+    snes.getKSP().getPC().setFactorSolverType("superlu_dist")
```

Comment:
PETSc 3.13 is out, and allow MUMPS to be configured for 64-bit indices.  Just waiting on petsc4py before updating the test system,

Reply:
Do you want to switch back to mumps, or just leave it on `superlu_dist`?

Reply:
Let's leave it as `superlu_dist` for now - gives use test coverage with 64-bit indices.